server/src/test/scala/services/crunch/ForecastPlanningToCSVDataTest.scala

```diff
@@ -245,12 +242,231 @@ class ForecastPlanningToCSVDataTest extends Specification {
+          |04:15,1,1,0,1,1,0,1,1,0
+          |04:30,1,1,0,1,1,0,1,1,0
+          |04:45,1,1,0,1,1,0,1,1,0
+          |05:00,1,1,0,1,1,0,1,1,0
+          |05:15,1,1,0,1,1,0,1,1,0
+          |05:30,1,1,0,1,1,0,1,1,0
+          |05:45,1,1,0,1,1,0,1,1,0
+          |06:00,1,1,0,1,1,0,1,1,0
+          |06:15,1,1,0,1,1,0,1,1,0
+          |06:30,1,1,0,1,1,0,1,1,0
+          |06:45,1,1,0,1,1,0,1,1,0
+          |07:00,1,1,0,1,1,0,1,1,0
+          |07:15,1,1,0,1,1,0,1,1,0
+          |07:30,1,1,0,1,1,0,1,1,0
+          |07:45,1,1,0,1,1,0,1,1,0
+          |08:00,1,1,0,1,1,0,1,1,0
+          |08:15,1,1,0,1,1,0,1,1,0
+          |08:30,1,1,0,1,1,0,1,1,0
+          |08:45,1,1,0,1,1,0,1,1,0
+          |09:00,1,1,0,1,1,0,1,1,0
+          |09:15,1,1,0,1,1,0,1,1,0
+          |09:30,1,1,0,1,1,0,1,1,0
+          |09:45,1,1,0,1,1,0,1,1,0
+          |10:00,1,1,0,1,1,0,1,1,0
+          |10:15,1,1,0,1,1,0,1,1,0
+          |10:30,1,1,0,1,1,0,1,1,0
+          |10:45,1,1,0,1,1,0,1,1,0
+          |11:00,1,1,0,1,1,0,1,1,0
+          |11:15,1,1,0,1,1,0,1,1,0
+          |11:30,1,1,0,1,1,0,1,1,0
+          |11:45,1,1,0,1,1,0,1,1,0
+          |12:00,1,1,0,1,1,0,1,1,0
+          |12:15,1,1,0,1,1,0,1,1,0
+          |12:30,1,1,0,1,1,0,1,1,0
+          |12:45,1,1,0,1,1,0,1,1,0
+          |13:00,1,1,0,1,1,0,1,1,0
+          |13:15,1,1,0,1,1,0,1,1,0
+          |13:30,1,1,0,1,1,0,1,1,0
+          |13:45,1,1,0,1,1,0,1,1,0
+          |14:00,1,1,0,1,1,0,1,1,0
+          |14:15,1,1,0,1,1,0,1,1,0
+          |14:30,1,1,0,1,1,0,1,1,0
+          |14:45,1,1,0,1,1,0,1,1,0
+          |15:00,1,1,0,1,1,0,1,1,0
+          |15:15,1,1,0,1,1,0,1,1,0
+          |15:30,1,1,0,1,1,0,1,1,0
+          |15:45,1,1,0,1,1,0,1,1,0
+          |16:00,1,1,0,1,1,0,1,1,0
+          |16:15,1,1,0,1,1,0,1,1,0
+          |16:30,1,1,0,1,1,0,1,1,0
+          |16:45,1,1,0,1,1,0,1,1,0
+          |17:00,1,1,0,1,1,0,1,1,0
+          |17:15,1,1,0,1,1,0,1,1,0
+          |17:30,1,1,0,1,1,0,1,1,0
+          |17:45,1,1,0,1,1,0,1,1,0
+          |18:00,1,1,0,1,1,0,1,1,0
+          |18:15,1,1,0,1,1,0,1,1,0
+          |18:30,1,1,0,1,1,0,1,1,0
+          |18:45,1,1,0,1,1,0,1,1,0
+          |19:00,1,1,0,1,1,0,1,1,0
+          |19:15,1,1,0,1,1,0,1,1,0
+          |19:30,1,1,0,1,1,0,1,1,0
+          |19:45,1,1,0,1,1,0,1,1,0
+          |20:00,1,1,0,1,1,0,1,1,0
+          |20:15,1,1,0,1,1,0,1,1,0
+          |20:30,1,1,0,1,1,0,1,1,0
+          |20:45,1,1,0,1,1,0,1,1,0
+          |21:00,1,1,0,1,1,0,1,1,0
+          |21:15,1,1,0,1,1,0,1,1,0
+          |21:30,1,1,0,1,1,0,1,1,0
+          |21:45,1,1,0,1,1,0,1,1,0
+          |22:00,1,1,0,1,1,0,1,1,0
+          |22:15,1,1,0,1,1,0,1,1,0
+          |22:30,1,1,0,1,1,0,1,1,0
+          |22:45,1,1,0,1,1,0,1,1,0
+          |23:00,1,1,0,1,1,0,1,1,0
+          |23:15,1,1,0,1,1,0,1,1,0
+          |23:30,1,1,0,1,1,0,1,1,0
+          |23:45,1,1,0,1,1,0,1,1,0""".stripMargin
+
+    result === expected
+  }
+
+  "Given a ForecastPeriod which includes a period that spans a timezone change from UTC to BST " +
```

Comment:
could we add another day here to make it clearer that the blank entries are only on the switchover day? does that makes sense? ie like the test going from BST to UTC